examples/imagenet/requirements.txt

```diff
@@ -0,0 +1,3 @@
+#additional requirements
+tensorflow-datasets==4.0.1
+tensorflow-gpu==2.2.0  #tensorflow-cpu also ok, but with gpu faster
```

Comment:
why is it faster with tf-gpu? :thinking: Here we are only using `tf.data` which runs in CPU, right?

Reply:
I believe this is because JPEG decoding is performed on the GPU. To be honest I have not tested it with elegy, I had noticed this when training with flax and simply adopted it here.